src/Kdyby/Events/DI/EventsExtension.php

```diff
@@ -313,8 +313,14 @@ private function autowireEvents(Nette\DI\ContainerBuilder $builder)
 					continue;
 				}
 			}
-			if ($def->getImplementType() === 'get') {
-				continue;
+			if (method_exists($def, 'getImplementMode')) { //BC (Nette 2.4)
```

Comment:
Nette 2.4 is new so "BC" comment is wrong here imho. Also I'd rather put the detection outside of the foreach. It's not really important in compile time but still.